### DISCUSSION TITLE
Addon names, sentence case or title case?

### DISCUSSION THREAD

**Discussion by Hans5958** [ScratchAddons/ScratchAddons#515] [Discussion]
For some time ago, we have a consensus to use sentence case ("Discuss button") instead of title case ("Discuss Button"). Now that I think about it, I think it is better to use title case due to the fact that it is more like a stand-out feature than a normal settings.

What do you guys think? Should we use title case, or keep using sentence case?

**Comment by mxmou** [comment]
> more like a stand-out feature than a normal settings

I think addons should look like normal settings.

**Comment by GrahamSH-LLK** [comment]
I personally like Title Case. It's the _title_ of the addon, after all.

**Reply by mxmou** [reply]
I don't think *Message count on extension icon* or *Highlight currently executing blocks* is a title of anything. It's just a name of a setting.

**Comment by WorldLanguages** [comment]
Keep in mind some addons are title case now, Scratch Messaging and Scratch Notifier.

**Reply by mxmou** [reply]
But those are names of existing extensions.

**Reply by Hans5958** [reply]
That doesn't matter. I could use the same argument to my "More links" addon, but nah.

**Comment by Hans5958** [comment]
@apple502j I see that you try to use title case, single-handedly, on the l10n files. Tell me why?

**Comment by apple502j** [comment]
@Hans5958 Where?

**Comment by lisa-wolfgang** [comment]
I think the addons with actual titles (like Scratch Messaging) should be given more generic names in sentence case. The former name could be included in the description with a note such as:
> This feature uses the Scratch Messaging addon by World_Languages and griffpatch.
